src/itest/kotlin/nl/lifely/zac/itest/ZaakRestServiceTest.kt

```diff
@@ -13,12 +13,12 @@ import io.kotest.core.spec.Order
 import io.kotest.core.spec.style.BehaviorSpec
 import io.kotest.matchers.shouldBe
 import io.kotest.matchers.shouldNotBe
+import net.atos.client.zgw.zrc.model.BetrokkeneType
```

Comment:
@hsiliev I am not sure what is the reason the `itest` are re-declaring enum values.

I removed `import nl.lifely.zac.itest.config.ItestConfiguration.BETROKKENE_TYPE_NATUURLIJK_PERSOON` but this might be wrong